Nodejs/Setup/NodejsToolsInstaller/NodejsToolsInstaller.wxs

```diff
@@ -153,11 +154,26 @@
         <CustomAction Id="VWDExpressSetup" Property="VWDEXPRESS_PATH" ExeCommand="/setup" Execute="deferred" Return="check" Impersonate="no" />
         <CustomAction Id="VWDExpressSetup_Rollback" Property="VWDEXPRESS_PATH" ExeCommand="/setup" Execute="rollback" Return="check" Impersonate="no" />
 
+        <!-- Custom Action and properties for installer telemetry-->
+        <Property Id="VSVersion" Value="$(var.VSTargetVersion)"></Property>
+        <Property Id="NTVSVersion" Value="$(var.ReleaseVersion)"></Property>
+        <Property Id="MSIVersion" Value="$(var.MsiVersion)"></Property>
```

Comment:
Names should be:
* VSVersion (what you have now)
* NtvsVersion
* MsiVersion

Acronyms > 2 letters should not be capitalized in Pascal case.
https://msdn.microsoft.com/en-us/library/vstudio/ms229043(v=vs.100).aspx